rules/integrations/aws/aws_eventbridge_rule_disabled_or_deleted.toml

```
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2021/10/17"
+maturity = "production"
+updated_date = "2021/10/17"
+integration = "aws"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when an user disables or deletes a EventBridge rule. An adversary user attempting to evade defenses by deleting or 
+disabling EventBridge rules.
```

```suggestion
Identifies when a user disabled or deleted an EventBridge rule. This activity can result in an unintended loss of
visibility in applications or breaking the flow with other AWS services.
```
Non security related